web/src/settings_invites.ts

```diff
@@ -206,13 +209,18 @@ function do_resend_invite({$row, invite_id}: {$row: JQuery; invite_id: string}):
         return;
     }
 
-    $resend_button.prop("disabled", true).text($t({defaultMessage: "Working…"}));
+    $resend_button.prop("disabled", true);
```

Comment:
Think button should be disabled only after the resent succeeded, since otherwise the button is disabled even if server returns an error.